packages/fints/src/segments/hicdb.ts

```diff
@@ -0,0 +1,60 @@
+import { SegmentClass } from "./segment";
+import { StandingOrder } from "../types";
+import { Parse } from "../parse";
+import {
+    document,
+    PaymentInstructionInformationSCT,
+    CreditTransferTransactionInformationSCT,
+    CustomerCreditTransferInitiationV03,
+} from "../pain-formats";
+
+export class HICDBProps {
+    public segNo: number;
+    public standingOrder: StandingOrder;
+}
+
+/**
+ * HICDB (SEPA-Dauerauftragsbestand rückmelden)
+ * Section C.10.2.3.4
+ */
+export class HICDB extends SegmentClass(HICDBProps) {
+    public type = "HICDB";
+
+    protected serialize(): string[][] { throw new Error("Not implemented."); }
+
+    protected deserialize(input: string[][]) {
+        const [
+            [],
+            [],
+            [ sepaMessage ],
+            [],
+            [ nextOrder, timeUnit, interval, orderDay, lastOrder ],
+        ] = input;
+
+        const jsonMessage: CustomerCreditTransferInitiationV03 =
+            Parse.xml<document>(sepaMessage).Document.CstmrCdtTrfInitn;
```

Comment:
Is it always guaranteed that `jsonMessage` has this type? Does the parser perform schema validation?

Reply:
I assumed that. No, the library does no schema validation.